sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/DigitalGyro.scala

```diff
@@ -0,0 +1,75 @@
+package com.lynbrookrobotics.potassium.sensors.imu
+
+import java.util
+
+import com.lynbrookrobotics.potassium.{PeriodicSignal, Signal}
+import squants.motion.Velocity
+
+abstract class DigitalGyro {
+  // Tick Period of the robot
+  final var TICK_PERIOD: Double = ??? //TODO set value based on final 2017 robot
```

Comment:
Instead of hard coding this, make it abstract and have anyone that extends this class take it as a constructor parameter.